go.mod

```diff
@@ -12,7 +12,7 @@ require (
 	github.com/spf13/cobra v1.1.3
 	github.com/spf13/pflag v1.0.5
 	golang.org/x/tools v0.1.2
-	k8s.io/apimachinery v0.21.2 // for `kubebuilder alpha config-gen`
+	k8s.io/apimachinery v0.22.1 // for `kubebuilder alpha config-gen`
```

Comment:
@camilamacedo86 just curious, why aren't we also bumping controller-runtime to 0.10.0 and controller-tools to 0.6.2 (the latest versions?)

Reply:
I create this PR just to update the tool. 

I did another PR to update the scaffolds. 
See: https://github.com/kubernetes-sigs/kubebuilder/pull/2340
However, we first need the https://github.com/kubernetes-sigs/controller-tools/pull/607 then, to do a release there and then we can update here.